test/e2e/puppeteer.js

```diff
@@ -113,6 +113,7 @@ const exceptionList = [
 	// Awaiting for WebGL backend support
 	'webgpu_clearcoat',
 	'webgpu_compute_audio',
+	"webgpu_compute_birds",
```

Comment:
It seems it was necessary to comment out the example because the WebGL 2 version is broken with this PR. The following WebGL warning occur:

> WebGL warning: drawArraysInstanced: Vertex fetch requires 147456, but attribs only supply 16384.